packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -382,6 +385,26 @@ class RenderEditable extends RenderBox {
     markNeedsLayout();
   }
 
+  /// How thick the cursor will be.
+  double get cursorWidth => _cursorWidth;
+  double _cursorWidth = 1.0;
+  set cursorWidth(double value) {
+    if (_cursorWidth == value)
+      return;
+    _cursorWidth = value;
+    markNeedsPaint();
```

Comment:
I think you still need layout in this specific setter since _caretPrototype seems to use it.